source/Mlos.Python/mlos/Optimizers/BayesianOptimizerConfigStore.py

```diff
@@ -18,7 +18,7 @@
                 HomogeneousRandomForestRegressionModel.__name__,
             ]),
             CategoricalDimension(name="experiment_designer_implementation", values=[ExperimentDesigner.__name__]),
-            DiscreteDimension(name="min_samples_required_for_guided_design_of_experiments", min=2, max=10000)
+            DiscreteDimension(name="min_samples_required_for_guided_design_of_experiments", min=2, max=100)

```

Comment:
A large number here turns the optimization process into random search, as the model is never refit. This can be better accomplished by setting fraction_random_suggestions = 1.